.gitignore

```diff
@@ -21,7 +21,8 @@ target
 /esvm
 .htpasswd
 .eslintcache
-plugins
+/plugins/
+/src/plugins/
```

Comment:
You're right. This must have been left over in my local repo from a previous time.